src/main/java/com/squareup/kotlinpoet/FileSpec.kt

```diff
@@ -99,8 +99,10 @@ class FileSpec private constructor(
   /** Writes this to `filer`.  */
   @Throws(IOException::class)
   fun writeTo(filer: Filer) {
-    val originatingElements = members.filterIsInstance<OriginatingElementsHolder>()
-        .flatMap(OriginatingElementsHolder::originatingElements)
+    val originatingElements = members.asSequence()
+        .filterIsInstance<OriginatingElementsHolder>()
+        .flatMap { it.originatingElements.asSequence() }
```

Comment:
Why are you creating a sequence here? The entire collection will always be consumed.

Reply:
`Sequence` has only one (this) variant of `flatMap`. What would you suggest instead?
I used `members.asSequence()` to avoid allocating temporary lists.

Reply:
Ah, I missed the original asSequence call on the outer chain. Makes sense.